ckanext/datajson/plugin.py

```diff
@@ -25,15 +30,14 @@ def update_config(self, config):
         # is called before after_map, in which we need the configuration directives
         # to know how to set the paths.
 
-        # TODO commenting out enterprise data inventory for right now
-        # DataJsonPlugin.route_edata_path = config.get("ckanext.enterprisedatajson.path", "/enterprisedata.json")
         DataJsonPlugin.route_enabled = config.get("ckanext.datajson.url_enabled", "True") == 'True'
-        DataJsonPlugin.route_path = config.get("ckanext.datajson.path", "/data.json")
+        DataJsonPlugin.route_path = config.get("ckanext.datajson.path", "/internal/data.json")
```

Comment:
Let's not change the default here. We should set it in our config...

Reply:
Spoke with James. Overriding the default would require an architectural overhaul. This needs to be discussed with upstream.